cinema/models.py

```diff
@@ -35,12 +39,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def custom_movie_image_path(instance, filename: str) -> pathlib.Path:
+    filename = (
+        f"{slugify(instance.title)}-{uuid.uuid4()}"
+        + pathlib.Path(filename).suffix
+    )
+    return pathlib.Path("uploads/movies/") / pathlib.Path(filename)
```

Comment:
you don't need to use `pathlib.Path(filename)`
It can be simplified to `pathlib.Path("uploads/movies/") / filename`